# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocprofiler-register/hotfix-no-cpack.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocprofiler-register-rocm-6.3.3/CMakeLists.txt.vanilla	2025-03-20 20:00:13.444219953 +0200
+++ rocprofiler-register-rocm-6.3.3/CMakeLists.txt	2025-03-20 20:00:15.351994459 +0200
@@ -113,6 +113,6 @@
     add_subdirectory(samples)
 endif()
 
-include(rocprofiler_register_config_packaging)
+#include(rocprofiler_register_config_packaging)
 
 rocprofiler_register_print_features()
